orchagent/qosorch.cpp

```diff
@@ -1014,7 +1014,7 @@ task_process_status QosOrch::handleQueueTable(Consumer& consumer)
 
     ref_resolve_status  resolve_result;
     // sample "QUEUE_TABLE:ETHERNET4:1"
-    tokens = tokenize(key, delimiter);
+    tokens = tokenize(key, delimiter2);
```

Comment:
The example doesn't match the code change here. Can you also update the example?